### PR TITLE
Add tricontour/tricontourf arguments(corner_mask, vmin vmax, antialiased, nchunk, hatches) documentation

### PR DESCRIPTION
**Pull request by AkM-2018**
## PR Summary
Related to #19431 

Some contour/contourf arguments which also work with tricontour/tricontourf are documented. 

## PR Checklist

- [N/A] Has pytest style unit tests (and `pytest` passes).
- [N/A] Is [Flake 8](https://flake8.pycqa.org/en/latest/) compliant (run `flake8` on changed files to check).
- [N/A] New features are documented, with examples if plot related.
- [N/A] Documentation is sphinx and numpydoc compliant (the docs should [build](https://matplotlib.org/devel/documenting_mpl.html#building-the-docs) without error).
- [x] Conforms to Matplotlib style conventions (install `flake8-docstrings` and run `flake8 --docstring-convention=all`).
- [N/A] New features have an entry in `doc/users/next_whats_new/` (follow instructions in README.rst there).
- [N/A] API changes documented in `doc/api/next_api_changes/` (follow instructions in README.rst there).

### CODE REVIEW

**Review by timhoffm — APPROVED** [review]

File: lib/matplotlib/tri/tricontour.py [hunk 1]
```diff
@@ -239,7 +251,22 @@ def _contour_args(self, args, kwargs):
...
+
+nchunk : int >= 0, optional
+    If 0, no subdivision of the domain.  Specify a positive integer to
+    divide the domain into subdomains of *nchunk* by *nchunk* quads.
+    Chunking reduces the maximum length of polygons generated by the
+    contouring algorithm which reduces the rendering workload passed
+    on to the backend and also requires slightly less RAM.  It can
+    however introduce rendering artifacts at chunk boundaries depending
+    on the backend, the *antialiased* flag and value of *alpha*.""")
+
```
Comment (timhoffm) [inline lib/matplotlib/tri/tricontour.py:269]:
```suggestion
```
There should only be two empty lines between functions.

This causes flake8 to complain: "E303 too many blank lines (3)".

**Review by jklymak — APPROVED** [review]
Good to go, but you should make the change @timhoffm  suggested...

**Comment by AkM-2018** [comment]
Removed the empty line.
Is there anything else that needs to be changed/added?

**Comment by jklymak** [comment]
Thanks @AkM-2018 !

**Comment by ianthomas23** [comment]
This has been prematurely merged.  There are no `corner_mask` or `nchunk` arguments to `tricontour(f)`.

**Comment by AkM-2018** [comment]
I'll correct that

**Comment by ianthomas23** [comment]
@AkM-2018 Thanks.  It will have to be another PR now that this has been merged.

**Comment by AkM-2018** [comment]
@ianthomas23 Please review the pull request #20171

**Pull request merged**